clean_project/index.py

```diff
@@ -31,17 +29,48 @@ def index(config_path, index_path):
                         index_words[word.lower()][source_file.name].append(i)
     else:
         raise ValueError('Empty configuration file')
-    with open(index_path, 'w') as index_file:
-        json.dump(index_words, index_file, sort_keys=True, indent=4)
     return index_words
 
 
+def build_index(index_words):
+    """
+    Insert the index in a specific database for store all data
+    :param index_words: Object which contains the built index
+    """
+    connection = None
+    try:
+        connection = psycopg2.connect(database='index_db', user='gianluca', password='password')
```

Comment:
you shouldn't put username and password in the code like this. these parameters should be read by flag arguments or by config file